requirements.txt

```diff
@@ -1,3 +1,4 @@
 fastapi >=0.100
 httpx
 uvicorn >=0.29.0
+asgi-lifespan >=2.1.0
```

Comment:
we need this because FastAPI lifespan won't be triggered with async client

> If your application relies on lifespan events, the AsyncClient won't trigger these events. To ensure they are triggered, use LifespanManager from [florimondmanca/asgi-lifespan](https://github.com/florimondmanca/asgi-lifespan#usage).